### DISCUSSION TITLE
Can Ubicloud use multi provider ?

### DISCUSSION THREAD

**Discussion by gensisco** [ubicloud/ubicloud#1902] [Q&A] [answered]
**Hello in my region there is two cheap cloud providers and i want to build my own cloud using this two providers does ubicloud support multi cloud ?
thanks**

**Comment by fdr** [comment, marked as answer]
Probably, so long as it has IPv6 available for the underlay networking, ideally statically routed (rather than having to tell the switch about each address via NDP). I've found this the least portable aspect of the program, but, generally, small adjustments to `learn_network.rb` can compensate for this.
